src/status_im/chat/events/input.cljs

```diff
@@ -413,6 +413,8 @@
  message-model/send-interceptors
  (fn [{{:keys [current-chat-id current-public-key] :as db} :db message-id :random-id current-time :now
        :as cofx} _]
+   (let [wut? nil]
+     (wut? :why?))
```

Comment:
also seems like leftover from local debugging session or something

Reply:
that's not leftover, i want to give QA person a chance to actually see that exception in instabug :)

Reply:
sorry, i didn't expect this way to trigger exception :)

Reply:
at least it was funny :D